content/en/docs/ops/ambient/install/istioctl-installation/index.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Install with Istioctl
+description: Install Istio in Ambient mode with Istioctl.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: yes
+---
+
+This guide shows you how to install Istio in ambient mode with Istioctl.
+Aside from following the demo in [Getting Started with Ambient Mode](/docs/ops/ambient/getting-started/),
+
+## Prerequisites
+
+1. Check the [Platform-Specific Prerequisites](/docs/ops/ambient/install/platform-prerequisites).
+
+1. Install the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.
+
+1.  Install the Kubernetes Gateway API CRDs, which don’t come installed by default on most Kubernetes clusters:
+
+    {{< text bash >}}
+    $ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
+      { kubectl kustomize "github.com/kubernetes-sigs/gateway-api/config/crd/experimental?ref={{< k8s_gateway_api_version >}}" | kubectl apply -f -; }
+    {{< /text >}}
+
+    {{< tip >}}
+    {{< boilerplate gateway-api-future >}}
+    {{< boilerplate gateway-api-choose >}}
+    {{< /tip >}}
+
+## Install
+
+{{< tabset category-name="config-api" >}}
+
+{{< tab name="Istio APIs" category-value="istio-apis" >}}
+
+{{< text bash >}}
+$ istioctl install --set profile=ambient --set "components.ingressGateways[0].enabled=true" --set "components.ingressGateways[0].name=istio-ingressgateway" --skip-confirmation
```

Comment:
Why include the ingressgateway chart? I think the best practice is to not use it.